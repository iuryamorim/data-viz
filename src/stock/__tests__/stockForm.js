import React from 'react'
import TestRenderer from 'react-test-renderer'

import StockForm from '../stockForm'


describe('StockForm', () =>  {
  it('render component', () =>  {
    const testRenderer = TestRenderer.create(<StockForm />).toJSON()
    expect(testRenderer).toMatchSnapshot()
    expect(testRenderer.type).toEqual('div')
  })
})
