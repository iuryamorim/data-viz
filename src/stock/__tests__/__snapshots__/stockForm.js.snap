// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StockForm render component 1`] = `
<div
  className="StockForm"
  role="form"
>
  <div>
    <div
      className="col-xs-12 col-sm-12 col-md-12 col-lg-12"
    >
      <div
        className="boxInline"
      >
        <input
          className="form-control"
          id="ticker"
          onKeyUp={[Function]}
          placeholder="Ticker"
        />
        <input
          className="form-control"
          id="start"
          onKeyUp={[Function]}
          placeholder="Start Date"
          type="date"
        />
        <input
          className="form-control"
          id="end"
          onKeyUp={[Function]}
          placeholder="End Date"
          type="date"
        />
        <select
          id="getFunction"
          name="getFunction"
        >
          <option
            value=""
          >
             --- Algorithms ---
          </option>
          <option
            value="dividends"
          >
            Dividends
          </option>
          <option
            value="close"
          >
            Close Price
          </option>
          <option
            value="recommendations"
          >
            Recommendations
          </option>
        </select>
        <button
          className="btn btn-info searchButton"
        >
          <i
            className="fa fa-search"
          />
        </button>
        <button
          className="btn btn-default undefined"
        >
          <i
            className="fa fa-close"
          />
        </button>
      </div>
    </div>
  </div>
</div>
`;
